README.rst

```diff
@@ -19,6 +19,10 @@ You can build higher level workflows, such as a Genomics portal that automates t
 
 Quick Start
 -----------
+**IMPORTANT**: you will need an **Amazon EC2 Key Pair** to be able to complete the following steps.
+Please see the `official AWS guide <https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html>`_
+if you need to create a Key Pair.
```

Comment:
I'd remove "if you need to create a Key Pair."

Reply:
Done